17 Observables/src/components/nameEdit/nameEdit.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export const NameEditComponent = (props: {userName: string, onChange: (name: string) => any}) => {
```

Comment:
I would remove all the components that are not related with the sample itself, just to get thing simpler, I would keep only the member list components, actions, reducers, what do you think?